app/models/uploaded_file.rb

```diff
@@ -1,22 +1,14 @@
 require "flex_deployment_client/middleware/uploaded_file_middleware"
-require "flex_deployment_client/requestor/uploaded_file_requestor"
+
 module FlexDeploymentClient
   class UploadedFile < ModelBase
-    def file=(value)
-      if /^file:\/\//.match?(value)
-        filename = value.match(/^file:\/\/(.*)$/)[1]
-        mime_type = MIME::Types.type_for(filename).first.content_type
-        bin_data = File.open(filename, "rb") { |f| f.read }
-        super("data:#{mime_type};base64,#{Base64.encode64(bin_data)}")
-      elsif /^http(s?):\/\//.match?(value)
-        self.file_url = value
-      else
-        super
-      end
-    end
```

Comment:
Turns out `value` is and has always been a hash here, not a string - so the only code path to ever be called was `super`.